docker/hive/Dockerfile

```diff
@@ -86,6 +86,7 @@ RUN chmod +x /wait
 COPY init-hive.sh /
```

Comment:
I know this is semi-related but why do we copy init-hive if we don't use it in microservices/production/hive-2.tf

Reply:
We use it in start-hive.sh

Reply:
Thanks